docs/device-agent/quickstart.md

```diff
@@ -8,60 +8,102 @@ meta:
       - flowfuse
 ---
 
-# Developing Flows on Your Device
+# Quick Start Guide: Device Agent
 
-## Add Device to FlowFuse
+This guide will walk you through the process of adding a device to FlowFuse, connecting it to the platform, and deploying your Node-RED flows remotely. FlowFuse's Device Agent helps unlock the power of your devices by allowing you to manage and deploy Node-RED flows running on those devices securely and remotely.
 
-### Prerequisites
+![FlowFuse Device Agent Workflow](./images/device-agent-workflow.png){data-zoomable}
+_Workflow to onboard new Edge Devices through the FlowFuse Platform and Device Agent_
 
-If you do not already have the device agent installed, follow the [install instructions](./install.md), otherwise, proceed to the next step.
+## Setup & Installation
 
-_NOTE: It is good practice to keep your Device Agent up to date. Update instructions are also provided in the link above._
+### Step 1: Install Device Agent (On Device)
 
-_NOTE: If you prefer to use the Device Agent WEB UI, follow the [Quick Start with Web UI](./quickstart-web-ui.md) guide._
+NodeJS version 18 is supported but version 20 or later is recommended. However, if necessary, device agent version 2.8 does support NodeJS versions 14 and 16.
+
+#### Linux/MacOS
+
+```bash
+sudo npm install -g @flowfuse/device-agent
+```
+
+#### Windows
+
+Issue the below command in an elevated command prompt:
+
+```bash
+npm install -g @flowfuse/device-agent
+```
+
+For full install instructions, follow the [install instructions here](/docs/device-agent/install.md).
```

Comment:
should this line not be in the parent section?

or at least repeated on the linux subsection?

currently, it is nested under the windows section.